pkgs/development/python-modules/datatable/default.nix

```
@@ -1,45 +1,59 @@
 { stdenv, lib, buildPythonPackage, fetchPypi, substituteAll, pythonOlder
+, pipInstallHook, writeText
 , blessed
 , docutils
 , libcxx
-, libcxxabi
 , llvm
-, openmp
 , pytest
 , typesentry
 }:
 
 buildPythonPackage rec {
   pname = "datatable";
-  version = "0.10.1";
+  version = "0.11.0";
   disabled = pythonOlder "3.5";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "3ce5257c0c4afa96e2b14ca47a0aaf73add195b11de48f4adda50b5ede927436";
+    sha256 = "1fvnval1gy38vx5jab9s5my07nj65msgmn4nwad2xxq03rqh5ihr";
```

```suggestion
    sha256 = "19c602711e00f72e9ae296d8fa742d46da037c2d3a2d254bdf68f817a8da76bb";
```
This is the full sha256 pypi's tarball download page: https://pypi.org/project/datatable/#copy-hash-modal-ba705dff-a3ae-4427-b710-a9742b18f5c9

---

ok - guess it's good for visibility.